python/objToJSON.c

```diff
@@ -295,10 +299,13 @@ static int Dict_iterNext(JSOBJ obj, JSONTypeContext *tc)
 
 static void Dict_iterEnd(JSOBJ obj, JSONTypeContext *tc)
 {
-  if (GET_TC(tc)->itemName)
-  {
-    Py_DECREF(GET_TC(tc)->itemName);
-    GET_TC(tc)->itemName = NULL;
+  if (GET_TC(tc)->itemName) {
+      Py_DECREF(GET_TC(tc)->itemName);
+      GET_TC(tc)->itemName = NULL;
+  }
```

Comment:
Accidental extra indentation here.

Reply:
ping @Adriandorr , this looks like the last remaining problem..?